### PR TITLE
Clize: Combine CLI

### PR DESCRIPTION
**Pull request by MoseleyS**
Combine CLI already had Clize decorators. This PR removes the argparser section and updates the acceptance tests to use the new argument names.

Testing:
 - [x] Ran tests and they passed OK

### CODE REVIEW

**Comment by benfitzpatrick** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 1
           

Complexity decreasing per file
==============================
+ lib/improver/cli/combine.py  -1
         
```


See the complete overview on [Codacy](https://app.codacy.com/app/metoppv_tech/improver/pullRequest?prid=4663882&bid=15613711)

File: lib/improver/cli/combine.py [hunk 1]
```diff
@@ -36,71 +36,6 @@
 from improver import cli
 
 
-def main(argv=None):
```
Comment (arh89) [inline lib/improver/cli/combine.py]: (@MoseleyS You'll need to remove the `if __name__` bit at the bottom as well)

File: lib/improver/tests/acceptance/test_combine.py [hunk 2]
```diff
@@ -50,7 +50,7 @@ def test_basic(tmp_path):
             "--new-name=cloud_area_fraction",
             low_cloud_path,
             medium_cloud_path,
-            output_path]
+            f"--output={output_path}"]
```
Comment (arh89) [inline lib/improver/tests/acceptance/test_combine.py]: It's fairly minor, but as a way of trying to standardise things a little more, we'd also like to try to have all of our arguments space separated, i.e: converting `--x=y` to `--x y`. Would it be possible to make this change here as well, please?

File: lib/improver/cli/combine.py (hunk 1 above)
Comment (MoseleyS) [reply lib/improver/cli/combine.py]: Done

File: lib/improver/tests/acceptance/test_combine.py (hunk 2 above)
Comment (MoseleyS) [reply lib/improver/tests/acceptance/test_combine.py]: I've changed this, but the result is a bit more messy as a blank space doesn't work, you have to separate the two halves of the argument into separate strings in the argument list. `"--output", f"{output_path"}`. Are you sure this is preferable?

**Comment by benfitzpatrick** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 1
           

Complexity decreasing per file
==============================
+ lib/improver/cli/combine.py  -1
         
```


See the complete overview on [Codacy](https://app.codacy.com/app/metoppv_tech/improver/pullRequest?prid=4663882&bid=15613711)

**Pull request merged**